README.md

```diff
@@ -196,4 +219,4 @@ source env_setup.sh
 
 ## Try it out
 
-Go to https://my_project_id.appspot.com  (replace my_project_id with your actual project ID)
+Go to https://my_project_id.uc.r.appspot.com  (replace my_project_id with your actual project ID)
```

Comment:
This is only the case if you use uscentral as your region.  Other regions get different tokens.  Google has published a list someplace, but at present I don't have that bookmarked.  Maybe a better instruction would be to tell users to go to the App Engine console page where the url is displayed in the upper right corner?